playbooks/bootstrap_server.yaml

```diff
@@ -10,6 +10,11 @@
   serial: "{{ batch_count | default('100%') }}"
   become: true
   gather_facts: false
+  vars_prompt:
```

Comment:
I would replace this with the `pause` module so that we just prompt when a reboot is required (recommended). See my example on the flush_handlers comment.